projects/example-app-e2e/src/integration/round-trip.spec.ts

```diff
@@ -38,7 +38,7 @@ context('Full round trip', () => {
   });
 
   it('is possible to add books', () => {
-    cy.get('bc-book-preview').eq(2).click();
+    cy.get('bc-book-preview').eq(1).click();
```

Comment:
🤦 seems like the google books API suddenly returns the same books more than once.